scripts/generator/graphql_generator/csharp/SDK/iOS/iOSWebCheckout.cs.erb

```diff
@@ -0,0 +1,43 @@
+namespace <%= namespace %>.SDK.iOS {
+#if !SHOPIFY_MONO_UNIT_TEST
```

Comment:
Should probably wrap this in `#if UNITY_IPHONE`, as well as where you end up using this in `Cart`